processing/data_collection/gazette/spiders/mg_itauna.py

```diff
@@ -0,0 +1,36 @@
+import datetime as dt
+from dateparser import parse
+from scrapy import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class MgItaunaSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3133808"
+    name = "mg_itauna"
+    allowed_domains = ["itauna.mg.gov.br"]
+    start_urls = ["https://www.itauna.mg.gov.br/portal/diario-oficial/"]
+
+    def parse(self, response):
+        url = self.start_urls[0] + "{i}/0/0/0/0/"
+        last_page = int(response.xpath("//select/option/text()")[-1].extract())
```

Comment:
**suggestion**: You can use `last()` as parameter to get the last `option` in the `select`. Doing that you avoid to 
`last_page = int(response.xpath("//select[@id='select']/option[last()]/text()").get())`

**nitpick**: Scrapy suggest that you use `get` (instead of `extract_first`) and `getall` (instead of `extract`) now. The old works, but it would be good if we keep consistency in the project.